src/test/java/seedu/address/testutil/TypicalPatients.java

```diff
@@ -23,36 +23,99 @@
  */
 public class TypicalPatients {
 
-    public static final Patient ALICE = new PatientBuilder().withName("Alice Pauline")
-            .withAddress("123, Jurong West Ave 6, #08-111").withEmail("alice@example.com")
+    public static final Patient ALICE = new PatientBuilder()
+            .withName("Alice Pauline")
+            .withAddress("123, Jurong West Ave 6, #08-111")
+            .withEmail("alice@example.com")
+            .withGender("female")
+            .withBirthday("23/09/2000")
+            .withIcNumber("T0032415E")
```

Comment:
I like how you adapted the TypicalPatients class to include our newly added fields!

Reply:
Agree with @thienmy0 on this point :)